web/wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -0,0 +1,151 @@
+<?php
+// If on Pantheon
+if( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ){
+	// Disable WordPress auto updates
+	if( ! defined('WP_AUTO_UPDATE_CORE')) {
+		define( 'WP_AUTO_UPDATE_CORE', false );
+	}
+
+	remove_action( 'wp_maybe_auto_update', 'wp_maybe_auto_update' );
+	// Remove the default WordPress core update nag
```

Comment:
:no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).